src/components/pages/Cases/CaseTable.js

```diff
@@ -9,8 +9,8 @@ import {
   CloseCircleOutlined,
   CheckCircleOutlined,
 } from '@ant-design/icons';
-import Save from '../../../styles/icons/save.svg';
-import Icon from '@ant-design/icons';
+// import Save from '../../../styles/icons/save.svg';
```

Comment:
You will definitely want to delete this if we are not going to use them!

Reply:
Agreed!